apps/studio/tests/integration/helpers/seed/index.ts

```diff
@@ -211,11 +211,32 @@ export const setupPageResource = async ({
   const { site, navbar, footer } = await setupSite(siteIdProp, !!siteIdProp)
   const blob = await setupBlob(blobIdProp)
 
+  let fallbackTitle = ""
+  let fallbackPermalink = ""
+  switch (resourceType) {
+    case "RootPage":
+      fallbackTitle = "Home"
+      fallbackPermalink = ""
+      break
+    case "CollectionPage":
+      fallbackTitle = "test collection page"
+      fallbackPermalink = "test-collection-page"
+      break
+    case "IndexPage":
+      fallbackTitle = "test index page"
+      fallbackPermalink = "test-index-page"
+      break
+    default:
+      fallbackTitle = "test page"
+      fallbackPermalink = "test-page"
+      break
+  }
```

Comment:
minor nit: probably can abstract as a function then return both fallback title and permalink as an object

Reply:
its only used within this method so i think it's fine, might be more complicated to extract out and refer to it elsewhere

Reply:
Hmm it's more of safety as `let` can be dangerous to use especially when there are more stuff to be done after this part. Using `const` here gives the assurance that it wouldn't accidentally be changed down the line, and the function does one thing by itself.

This is more of a best practices thing but given this is a helper function I leave it up to you.